Project-Aurora/Project-Aurora/Project-Aurora.csproj

```diff
@@ -98,18 +98,18 @@
     <PackageReference Include="MouseKeyHook.NetCore" Version="5.6.1.1" />
     <PackageReference Include="NAudio" Version="2.1.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="NLog" Version="4.7.10" />
-    <PackageReference Include="Octokit" Version="2.0.1" />
-    <PackageReference Include="PropertyChanged.Fody" Version="4.1.0" />
+    <PackageReference Include="NLog" Version="5.0.4" />
+    <PackageReference Include="Octokit" Version="3.0.0" />
    <!-- <PackageReference Include="OpenRGB.NET" Version="1.7.0" /> -->
+    <PackageReference Include="PropertyChanged.Fody" Version="4.1.0" />
     <PackageReference Include="RGB.NET.Devices.Asus" Version="2.0.0-prerelease.12" />
     <PackageReference Include="RGB.NET.Devices.CoolerMaster" Version="2.0.0-prerelease.12" />
     <PackageReference Include="RGB.NET.Devices.Corsair" Version="2.0.0-prerelease.12" />
     <PackageReference Include="RGB.NET.Devices.Logitech" Version="2.0.0-prerelease.12" />
     <PackageReference Include="RGB.NET.Devices.SteelSeries" Version="2.0.0-prerelease.12" />
     <PackageReference Include="SemanticVersioning" Version="2.0.2" />
     <PackageReference Include="SharpDX.Direct3D11" Version="4.2.0" />
-    <PackageReference Include="SharpDX.RawInput" Version="4.0.1" />
+    <PackageReference Include="SharpDX.RawInput" Version="4.2.0" />
```

Comment:
this breaks all input capture

Reply:
I guess there was an issue introduced with their interop with RawInput that was never fixed. I'll revert and add a link for future maintainers
https://github.com/sharpdx/SharpDX/issues/1050